### ISSUE TITLE
Warning dialog for converting changes to suggestions can be incorrect and vague

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode-pull-request-github#6263]
1. Check out a PR
2. Edit a line within a commenting range and a line not in a commenting range
3. From the SCM view, use the gift icon to convert your changes to PR suggestion comments
4. You'll see a dialog. It should indicate the number of converted suggestions that succeeded. 

// cc @benibenj

**Comment by amunger** [comment]
I'm not able to find the referenced icon
![Image](https://github.com/user-attachments/assets/364cfbc5-f8c4-4ace-9ced-974ece6d7158)

**Comment by TylerLeonhardt** [comment]
I'm pretty sure it's now this icon?
![Image](https://github.com/user-attachments/assets/696ff145-bf24-4876-a6a7-7dd50d59bdf2)

I tried that and I got this:
![Image](https://github.com/user-attachments/assets/4aa2d8a3-17de-4490-b56f-9d3e89dd8e05)

Even though my change was in the comment range as you can see.

**Comment by alexr00** [comment]
@TylerLeonhardt I'm not able to reproduce that

![Image](https://github.com/user-attachments/assets/54b16e3e-86b5-4c70-997d-78cdab981ccf)

If you still see it, can you share the exact PR that you see this with?

**Comment by TylerLeonhardt** [comment]
I still repro:
![Image](https://github.com/user-attachments/assets/d08c0e81-e6f1-41b3-9427-482b6c28ece4)

Steps:
* Open this PR: https://github.com/microsoft/vscode/pull/232058
* Open the inlineEditsView.ts
* Add `// A very important comment` in the range and then right outside of the range like so:
    ![Image](https://github.com/user-attachments/assets/9dd07926-7ff1-4add-9c35-3f4c9b4fe4e6)
* Do the create pull request suggestions

**Comment by alexr00** [comment]
Thank you! I can repro with that example!

**Comment by roblourens** [comment]
Getting this

![Image](https://github.com/user-attachments/assets/fdce4f96-8988-4abc-9c5c-1fda6655c831)

I believe this should be "1 of 2"

```
2024-12-06 15:00:41.254 [info] Review+1> Switch to Pull Request #232058 - start
2024-12-06 15:00:41.366 [info] PullRequestGitHelper> Branch origin/hediet/b/only-lark doesn't exist on local disk yet.
2024-12-06 15:00:41.366 [info] PullRequestGitHelper> Fetch tracked branch refs/remotes/origin/hediet/b/only-lark
2024-12-06 15:00:42.855 [error] Review+1> Checkout failed #{"stdout":"","stderr":"fatal: couldn't find remote ref hediet/b/only-lark\n","exitCode":128,"gitCommand":"fetch","gitArgs":["fetch","origin","hediet/b/only-lark"]}
2024-12-06 15:00:42.855 [info] Review+1> Validate state in progress
2024-12-06 15:00:42.855 [info] Review+1> Validating state...
2024-12-06 15:00:42.862 [info] Found GitHub remote for folder /Users/roblou/code/vscode
2024-12-06 15:00:42.863 [info] Trying to use globalState for mentionableUsers.
2024-12-06 15:00:42.863 [info] Using in-memory cached assignable users.
2024-12-06 15:00:42.888 [info] Review+1> Found matching pull request metadata for current branch roblou/visible-mackerel. Repo: microsoft/vscode PR: 235518
2024-12-06 15:00:42.888 [info] Review+1> current branch roblou/visible-mackerel is associated with pull request #235518
2024-12-06 15:00:42.888 [info] Review+1> Resolving pull request
2024-12-06 15:00:42.888 [info] FolderRepositoryManager+1> Found GitHub repo for pr #235518: yes
2024-12-06 15:00:42.897 [info] Using globalState mentionableUsers for 1.
2024-12-06 15:00:43.412 [info] FolderRepositoryManager+1> Found GitHub pr repo for pr #235518: yes
2024-12-06 15:00:43.412 [info] GitHubRepository+1> Fetch branch release/1.96 - enter
2024-12-06 15:00:43.742 [info] GitHubRepository+1> Fetch branch release/1.96 - done: true
2024-12-06 15:00:43.742 [info] Review+1> Resolved PR #235518, state is 1
2024-12-06 15:00:44.360 [info] Found matching pull request metadata on GitHub for current branch roblou/visible-mackerel. Repo: microsoft/vscode PR: 235518
2024-12-06 15:00:44.360 [info] PullRequestGitHelper> associate roblou/visible-mackerel with Pull Request #235518
2024-12-06 15:01:29.891 [info] Review+1> Validate state in progress
2024-12-06 15:01:29.891 [info] Review+1> Validating state...
2024-12-06 15:01:29.926 [info] Found GitHub remote for folder /Users/roblou/code/vscode
2024-12-06 15:01:29.927 [info] Using in-memory cached mentionable users.
2024-12-06 15:01:29.927 [info] Using in-memory cached assignable users.
2024-12-06 15:01:29.954 [info] Review+1> No matching pull request metadata found locally for current branch hediet/b/only-lark
2024-12-06 15:01:30.516 [info] Found matching pull request metadata on GitHub for current branch hediet/b/only-lark. Repo: microsoft/vscode PR: 232058
2024-12-06 15:01:30.516 [info] PullRequestGitHelper> associate hediet/b/only-lark with Pull Request #232058
2024-12-06 15:01:31.683 [info] Review+1> Found matching pull request metadata for current branch hediet/b/only-lark. Repo: microsoft/vscode PR: 232058
2024-12-06 15:01:31.683 [info] Review+1> current branch hediet/b/only-lark is associated with pull request #232058
2024-12-06 15:01:31.683 [info] Review+1> Resolving pull request
2024-12-06 15:01:31.683 [info] FolderRepositoryManager+1> Found GitHub repo for pr #232058: yes
2024-12-06 15:01:32.174 [info] FolderRepositoryManager+1> Found GitHub pr repo for pr #232058: yes
2024-12-06 15:01:32.174 [info] GitHubRepository+1> Fetch branch main - enter
2024-12-06 15:01:32.503 [info] GitHubRepository+1> Fetch branch main - done: true
2024-12-06 15:01:32.503 [info] Review+1> Resolved PR #232058, state is 2
2024-12-06 15:01:32.984 [info] Found matching pull request metadata on GitHub for current branch hediet/b/only-lark. Repo: microsoft/vscode PR: 232058
2024-12-06 15:01:32.984 [info] PullRequestGitHelper> associate hediet/b/only-lark with Pull Request #232058
2024-12-06 15:01:33.407 [info] Review+1> This PR is closed
2024-12-06 15:01:33.407 [info] PullRequestTree> Removing PR #235518 from tree
2024-12-06 15:01:33.447 [info] PullRequestTree> Comments have changed, refreshing Files node
2024-12-06 15:01:33.447 [info] PullRequestTree> Comments have changed, refreshing Commits node
2024-12-06 15:02:57.985 [info] Review+1> Switch to Pull Request #235529 - start
2024-12-06 15:02:58.040 [info] PullRequestGitHelper> Tried to checkout hediet/b/only-lark, but branch is already checked out.
2024-12-06 15:02:58.551 [info] Review+1> Switch to Pull Request #235529 - done
2024-12-06 15:02:58.551 [info] Review+1> Validate state in progress
2024-12-06 15:02:58.551 [info] Review+1> Validating state...
2024-12-06 15:02:58.553 [info] Found GitHub remote for folder /Users/roblou/code/vscode
2024-12-06 15:02:58.553 [info] Using in-memory cached assignable users.
2024-12-06 15:02:58.593 [info] Review+1> Found matching pull request metadata for current branch hediet/b/only-lark. Repo: microsoft/vscode PR: 232058
2024-12-06 15:02:58.593 [info] Review+1> current branch hediet/b/only-lark is associated with pull request #232058
2024-12-06 15:02:58.593 [info] Review+1> Resolving pull request
2024-12-06 15:02:58.593 [info] FolderRepositoryManager+1> Found GitHub repo for pr #232058: yes
2024-12-06 15:02:59.234 [info] FolderRepositoryManager+1> Found GitHub pr repo for pr #232058: yes
2024-12-06 15:02:59.234 [info] GitHubRepository+1> Fetch branch main - enter
2024-12-06 15:02:59.766 [info] GitHubRepository+1> Fetch branch main - done: true
2024-12-06 15:02:59.766 [info] Review+1> Resolved PR #232058, state is 2
2024-12-06 15:03:00.533 [info] Found matching pull request metadata on GitHub for current branch hediet/b/only-lark. Repo: microsoft/vscode PR: 235529
2024-12-06 15:03:00.533 [info] PullRequestGitHelper> associate hediet/b/only-lark with Pull Request #235529
2024-12-06 15:03:00.843 [info] Review+1> Resolving pull request
2024-12-06 15:03:00.843 [info] FolderRepositoryManager+1> Found GitHub repo for pr #235529: yes
2024-12-06 15:03:01.489 [info] FolderRepositoryManager+1> Found GitHub pr repo for pr #235529: yes
2024-12-06 15:03:01.489 [info] GitHubRepository+1> Fetch branch main - enter
2024-12-06 15:03:02.119 [info] GitHubRepository+1> Fetch branch main - done: true
2024-12-06 15:03:02.120 [info] Review+1> Fetching pull request data
2024-12-06 15:03:02.120 [info] PullRequestTree> Adding PR #235529 to tree
2024-12-06 15:03:02.188 [info] Review+1> Register comments provider
2024-12-06 15:03:02.189 [info] Review+1> Register in memory content provider
2024-12-06 15:03:02.283 [info] PullRequestTree> Creating file and commit nodes for PR #235529
2024-12-06 15:03:02.284 [info] PullRequestTree> Getting children for Files node
2024-12-06 15:03:02.940 [info] PullRequestTree> Local files have changed, refreshing Files node
2024-12-06 15:03:02.940 [info] PullRequestTree> Local files have changed, refreshing Files node
2024-12-06 15:03:02.954 [info] Review+1> Display pull request status bar indicator.
2024-12-06 15:03:02.954 [info] Review+1> Using focus mode = true.
2024-12-06 15:03:02.954 [info] Review+1> State validation silent = false.
2024-12-06 15:03:02.954 [info] Review+1> PR show should show = false.
2024-12-06 15:03:03.009 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/model/inlineCompletionsModel.ts
2024-12-06 15:03:03.009 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/view/inlineEdits/inlineEditsIndicatorView.ts
2024-12-06 15:03:03.009 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/view/inlineEdits/inlineEditsView.ts
2024-12-06 15:03:03.052 [info] FolderRepositoryManager+1> Found GitHub repo for pr #235529: yes
2024-12-06 15:03:03.140 [info] PullRequestTree> Getting children for Files node
2024-12-06 15:03:03.140 [info] PullRequestTree> Got all children for Files node
2024-12-06 15:03:03.400 [info] PullRequestTree> Review threads have changed, refreshing Files node
2024-12-06 15:03:03.400 [info] PullRequestTree> Review threads have changed, refreshing Commits node
2024-12-06 15:03:03.400 [info] PullRequestTree> Comments have changed, refreshing Files node
2024-12-06 15:03:03.400 [info] PullRequestTree> Comments have changed, refreshing Commits node
2024-12-06 15:03:03.400 [info] PullRequestTree> Review threads have changed, refreshing Files node
2024-12-06 15:03:03.400 [info] PullRequestTree> Review threads have changed, refreshing Commits node
2024-12-06 15:03:03.412 [info] PullRequestTree> Getting children for Files node
2024-12-06 15:03:03.412 [info] PullRequestTree> Got all children for Files node
2024-12-06 15:03:03.606 [info] PullRequestTree> Getting children for Files node
2024-12-06 15:03:03.606 [info] PullRequestTree> Got all children for Files node
2024-12-06 15:03:03.929 [info] FolderRepositoryManager+1> Found GitHub pr repo for pr #235529: yes
2024-12-06 15:03:16.355 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/view/inlineEdits/inlineEditsIndicatorView.ts
2024-12-06 15:03:16.355 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/model/inlineCompletionsModel.ts
2024-12-06 15:03:30.865 [info] FolderRepositoryManager+1> Found GitHub repo for pr #235529: yes
2024-12-06 15:03:30.899 [info] PullRequestTree> Review threads have changed, refreshing Files node
2024-12-06 15:03:30.899 [info] PullRequestTree> Review threads have changed, refreshing Commits node
2024-12-06 15:03:31.480 [info] PullRequestTree> Review threads have changed, refreshing Files node
2024-12-06 15:03:31.480 [info] PullRequestTree> Review threads have changed, refreshing Commits node
2024-12-06 15:03:31.581 [info] FolderRepositoryManager+1> Found GitHub pr repo for pr #235529: yes
2024-12-06 15:04:25.015 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/view/inlineEdits/inlineEditsIndicatorView.ts
2024-12-06 15:04:25.016 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/model/inlineCompletionsModel.ts
2024-12-06 15:04:27.237 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/view/inlineEdits/inlineEditsIndicatorView.ts
2024-12-06 15:04:27.238 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/model/inlineCompletionsModel.ts
2024-12-06 15:04:48.425 [info] FolderRepositoryManager+1> Found GitHub repo for pr #235529: yes
2024-12-06 15:04:49.088 [info] FolderRepositoryManager+1> Found GitHub pr repo for pr #235529: yes
2024-12-06 15:04:50.311 [info] PullRequestTree> Local files have changed, refreshing Files node
2024-12-06 15:04:50.311 [info] PullRequestTree> Local files have changed, refreshing Files node
2024-12-06 15:04:50.923 [info] PullRequestTree> Review threads have changed, refreshing Files node
2024-12-06 15:04:50.923 [info] PullRequestTree> Review threads have changed, refreshing Commits node
2024-12-06 15:04:50.924 [info] PullRequestTree> Comments have changed, refreshing Files node
2024-12-06 15:04:50.924 [info] PullRequestTree> Comments have changed, refreshing Commits node
2024-12-06 15:04:52.795 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/view/inlineEdits/inlineEditsIndicatorView.ts
2024-12-06 15:04:52.796 [info] GitContentFileSystemProvider> Getting change model (file:///Users/roblou/code/vscode) content for commit 61ba31d9af8d633bfdbede3286c30922feb0bfc3 and path /Users/roblou/code/vscode/src/vs/editor/contrib/inlineCompletions/browser/model/inlineCompletionsModel.ts
```

**Comment by alexr00** [comment]
I broke this with https://github.com/microsoft/vscode-pull-request-github/pull/6504.
